src/Spice86/Converters/LinearAddressToHexStringConverter.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Avalonia.Data;
+using Avalonia.Data.Converters;
+
+using Spice86.Models.Debugging;
+
+using System.Globalization;
+
+namespace Spice86.Converters;
+
+public class LinearAddressToHexStringConverter : IValueConverter {
```

Comment:
This class appears to be unused. Is there a future planned use for it?

Reply:
No, this has been replaced with StringFormat={}{0:X} in XAML